src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -118,8 +123,12 @@ def _search_role_definitions(definitions_client, name, scope, custom_role_only=F
     return roles
 
 
-def create_role_assignment(cmd, role, assignee, resource_group_name=None, scope=None):
-    return _create_role_assignment(cmd.cli_ctx, role, assignee, resource_group_name, scope)
+def create_role_assignment(cmd, role, assignee=None, assignee_object_id=None, resource_group_name=None, scope=None):
+    if bool(assignee) == bool(assignee_object_id):
+        raise CLIError('usage error: --assignee STRING | --assignee-object-id GUID')
+    resolve_assignee = not assignee_object_id
```

Comment:
nit: Can this assignment just be rolled into the parameter assignment on line 131?